cmd/list-app-owners.go

```diff
@@ -78,13 +80,13 @@ func listAppOwners(ctx context.Context, client client.AzureClient, apps <-chan a
 					}
 					count = 0
 				)
-				for item := range client.ListAzureADAppOwners(ctx, app.Data.Id, "", "", "", nil) {
+				for item := range client.ListAzureADAppOwners(ctx, app.Data.Id, params) {
 					if item.Error != nil {
 						log.Error(item.Error, "unable to continue processing owners for this app", "appId", app.Data.AppId)
 					} else {
 						appOwner := models.AppOwner{
 							Owner: item.Ok,
-							AppId: item.AppId,
+							AppId: app.Data.Id,
```

Comment:
Please assert there are no differences between `app.Data.Id` and `app.Data.AppId`. IIRC there is some nuance between an app's object ID and it's appID depending on the environment. For example, is the app in one tenant an app native to that tenant or is it a pointer to an app in another tenant?

It may be fine but we should verify.

Reply:
I went with `app.Data.Id` b/c that is the variable supplied into the `ListAzureADAppOwners` and that particular argument was then directly supplied in as `AppId` without manipulation.

Caller before:
<img width="639" alt="image" src="https://github.com/user-attachments/assets/6a60a26a-d021-41a1-adc6-7e15e70ff8f9">
Use within function:
<img width="1214" alt="image" src="https://github.com/user-attachments/assets/1815e025-7afc-41dd-8e85-c2e99adefb13">
...
<img width="578" alt="image" src="https://github.com/user-attachments/assets/d7c43f91-40e1-40aa-9626-8d9404af8b87">